articles/liquity/Asset Risk Assessment - bLUSD.md

```diff
@@ -0,0 +1,203 @@
+# Asset Risk Assessment bLUSD
+### Chicken Bonds are designed to generate amplified yields for the users and protocol-owned liquidity that can be used to strengthen the subjected asset.
+
+
+# Liquity protocol overview
+
+*[Resources]*
+
+*[Official Liquity Documentation](https://docs.liquity.org/)*
+
+*[Liquity: Decentralized Borrowing Platform](https://docsend.com/view/bwiczmy) (Whitepaper)*
+
+Liquity is a decentralized borrowing protocol that allows interest-free loans against ETH as collateral. Loans are paid out in LUSD and need to maintain a minimum collateral ratio (CR) of 110%. The loans are secured by a Stability Pool containing LUSD that caters the liquidations.
+
+Borrowers on Liquity open and maintain a vault called a “Trove” for their debt positions. The Stability Pool is a source of liquidity that is used to repay debt from liquidated Troves. When any Trove is liquidated, an amount of LUSD corresponding to the remaining debt of the Trove is burned from the Stability Pool’s balance to repay the Trove’s debt. In exchange, the entire collateral from the Trove is transferred to the Stability Pool. The Stability Pool is funded by users (called Stability Providers) transferring LUSD into it. Over time, Stability Providers lose a pro-rata share of their LUSD deposits, while gaining a pro-rata share from liquidated Troves.
+
+![Demand Supply flow-diagram for Liquity protocol](https://lh5.googleusercontent.com/MnSjLTcQzoWsjZgKvLYA67mF4uG8odRIfTtOE4U24lZpPx-vQ5uKF_Fn1kAlHYE18coYH4vh-bB5PbdMOYufDmg50a1J7cdxVBip4WbVQCaSGPEmUtt7nT79ggQjqfJMgtt7Vv8nkbeCt10YJl0hYRXx54cPBeEc8XBNaVk92_5fckd47B0axGZAeiGYRQ)
+
+Liquidation rewards are evenly distributed to the LUSD deposited in the Stability Pool. When the ETH price drops quickly, heavy liquidations might get triggered. If people are not confident with exposure to ETH, they might not deposit their LUSD in the stability pool. When the stability pool is empty, the debt obligation is transferred to other LUSD debt holders, reducing their collateral ratio. In the best case, the debt is transferred at a 110% collateral ratio, but in a flash crash, the ratio might drop lower than 110% reducing the collateral ratio for other debt holders.
+
+In an undesirable scenario where the Stability Pool has no funds to facilitate the liquidations, the debt obligation from a Trove that is deemed to be liquidated might get transferred to other LUSD debt holders. Transferring this debt obligation will reduce the health factor (collateral ratio) of the remaining Troves making them vulnerable to liquidation if the ETH price chooses to go down.
```

Comment:
"chooses to go down" -> "goes down"
(or "in the event of an adverse ETH price movement")